packages/spright-components/rollup.config.js

```diff
@@ -0,0 +1,60 @@
+import commonJS from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
+import sourcemaps from 'rollup-plugin-sourcemaps';
+import terser from '@rollup/plugin-terser';
+import replace from '@rollup/plugin-replace';
+import json from '@rollup/plugin-json';
+import nodePolyfills from 'rollup-plugin-polyfill-node';
```

Comment:
Was initially thinking we could remove things that are, i.e. table or rich-text specific but this build does need to be able to build all-components from nimble so all this stuff stays 👍

Reply:
Re-opening, guess things were removed from the roll-up. As stated since we are pulling in all nimble-components need the stuff from there.

The circular dependency stuff is missing: https://github.com/ni/nimble/blob/main/packages/nimble-components/rollup.config.js#L21

Resulting in build warnings: https://github.com/ni/nimble/actions/runs/8759928156/job/24043951459#step:12:1472

Reply:
FYI @m-akinc this comment is still open